Causality_analysis.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+
+subjects_dir = '/home/qdong/freesurfer/subjects/'
```

Comment:
@dongqunxi can you please remove the individual specific paths and maybe let the functions also take subjects_dir as input?